cloud/google/gce.py

```diff
@@ -141,7 +147,7 @@
 
 requirements:
     - "python >= 2.6"
-    - "apache-libcloud >= 0.13.3"
+    - "apache-libcloud >= 0.17.0"
```

Comment:
no, not 'yet'. but this documents the code so if users see this they'll feel they HAVE to upgrade
```   - "apache-libcloud >= 0.13.3, >= 0.17.0 if using JSON credentials"```
might be a better change